### PR TITLE
Add Sell custom objects app example

### PR DESCRIPTION
**Pull request by darrenchan**
@zendesk/documentation @beetlehope @nasaabg

### CODE REVIEW

File: v2/sell/custom-objects-app/README.md [hunk 1]
```diff
@@ -0,0 +1,508 @@
+# Custom objects app
```
Comment (darrenchan) [inline v2/sell/custom-objects-app/README.md:1]: @beetlehope @nasaabg  can you review this readme file? Thanks!

File: v2/sell/custom-objects-app/README.md [hunk 2]
```diff
@@ -0,0 +1,508 @@
...
+To use the app, complete the tasks as described in the following sections.
+
+### Enabling custom objects
+
+Custom objects must be enabled by an administrator in Zendesk Support. If you're not an admin, ask one to enable them for you. For more information, see [Enablng custom objects](https://support.zendesk.com/hc/en-us/articles/360037716253-Sunshine-custom-objects-guide-for-admins#topic_fk5_wyl_mjb).
+
+### Installing dependencies
+
+Run the following commands to install the necessary packages:
+` $ npm install $ npm install node-fetch `
```
Comment (beetlehope) [inline v2/sell/custom-objects-app/README.md:47]:
```suggestion
` $ npm install` 
` $ npm install node-fetch`
```

File: v2/sell/custom-objects-app/README.md [hunk 3]
```diff
@@ -0,0 +1,508 @@
...
+
+If you're interested in becoming a Zendesk developer partner, you can convert a trial account into a sponsored Zendesk Support account. See [Getting a trial or sponsored account for development](https://developer_v2.zendesk.com/documentation/developer-tools/getting-started/getting-a-trial-or-sponsored-account-for-development/).
+
+## Getting started
+
+To use the app, complete the tasks as described in the following sections.
+
+### Enabling custom objects
+
+Custom objects must be enabled by an administrator in Zendesk Support. If you're not an admin, ask one to enable them for you. For more information, see [Enablng custom objects](https://support.zendesk.com/hc/en-us/articles/360037716253-Sunshine-custom-objects-guide-for-admins#topic_fk5_wyl_mjb).
```
Comment (beetlehope) [inline v2/sell/custom-objects-app/README.md:42]:
```suggestion
Custom objects must be enabled by an administrator in Zendesk Support. If you're not an admin, ask one to enable them for you. For more information, see [Enabling custom objects](https://support.zendesk.com/hc/en-us/articles/360037716253-Sunshine-custom-objects-guide-for-admins#topic_fk5_wyl_mjb).
```

**Review by beetlehope — APPROVED** [review]
Great job!

**Review by nasaabg — APPROVED** [review]
Readme looks very professional! Good job @darrenchan

**Pull request merged**